addons/mail/static/src/backend_components/kanban_field_activity_view/kanban_field_activity_view_container.js

```diff
@@ -59,7 +59,7 @@ export class KanbanFieldActivityViewContainer extends Component {
         const kanbanFieldActivityView = messaging.models['KanbanFieldActivityView'].insert({
             id: this.kanbanFieldActivityViewId,
             thread: {
-                activities: this.props.value.records.map(activityData => {
```

Comment:
Indeed fix could be done in v16, I'm not sure if there is an actual issue related to it, but potentially yes.

Reply:
I wouldn't bother with fixing in 16.0 if we don't see any functional issue

Reply:
Well if there is a props change obviously the value inserted will be wrong (it will insert the previous value, always one step behind).
I'm not sure when there is a props change, but if there ever is the issue is obvious from code.